src/schema.js

```diff
@@ -93,8 +93,24 @@ const postSchema = Joi.object({
 	published: Joi.boolean()
 })
 
+const updatePostSchema = Joi.object({
+	userId: Joi.number()
+		.required(),
+	title: Joi.string()
+		.min(3)
+		.max(30)
+		.required(),
+	image: Joi.string()
+		.uri()
+		.required(),
+	content: Joi.string()
+		.required(),
+	published: Joi.boolean()
+})
+
 module.exports = {
 	'/create-user': authSchema,
```

Comment:
@beblicarl based on the limitations of using the routes, will just be exporting the schema .eg. 

```suggestion
	authSchema,
```